packages/aws-cdk/README.md

```diff
@@ -202,6 +203,10 @@ $ cdk deploy --no-rollback
 $ cdk deploy -R
 ```
 
+If a deployment fails you can update your code and immediately retry the
+deployment from the point of failure. If you would like to explicitly roll back a failed, paused deployment,
+use `cdk rollback`.
+
 NOTE: you cannot use `--no-rollback` for any updates that would cause a resource replacement, only for updates
```

Comment:
I don't know. My guess is that the storage that holds `(old resource, new resource)` can't outlast a single workflow execution. I guess there's no reason it couldn't be part of the long-term stack state, so that a rollback could delete the new resource and stabilize on the old one. But that's not how it's done.

Reply:
I tried it and it worked...I might have missed something though. Anyway, we can punt this.

Reply:
It would surprise me, since all the CFN docs say it doesn't work. Doesn't mean the docs aren't out of date though.